WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -13,4 +13,38 @@ def competition_message_for_user(competition, user)
                 end
     messages.join(' ')
   end
+
+  def announced_class(distance_announced)
+    if distance_announced <= 21
+      "alert-danger"
+    elsif distance_announced <= 28
```

Comment:
Is it actually possible for this `elsif` to ever hit? It looks like we'll always land in the previous if before going into this.